platform/broadcom/sonic-platform-modules-inventec/d7332/modules/lpc_ich.c

```diff
@@ -0,0 +1,1131 @@
+/*
+ *  lpc_ich.c - LPC interface for Intel ICH
```

Comment:
already in kernel, why do we need it?

Reply:
"gpio_version" is required but missing under [LPC_LPT] in current kernel version.
Since lpc_ich.c is a free software, we added the "gpio_version" for [LPC_LPT] in lpc_ich.c to make it work in our platform.

![image](https://user-images.githubusercontent.com/44458761/67060811-e4ef8d80-f190-11e9-9a30-367c67d48146.png)